### DISCUSSION TITLE
HTTP APIでincidentを作成しても画面に表示されない

### DISCUSSION THREAD

**Discussion by redfoxnodareiki** [topotal/waroom-roadmap#15] [Bug Report]
DocumentのHTTP IntegrationのcurlサンプルをIntegration keyを指定して実行し、uuid, created_at等レスポンス上は成功しているように見受けられますが、incidentsには表示されませんでした。(Unresolved, Resolvedの両リスト共に)
https://beta.waroom.com/docs/incoming_integration
```
curl --location --request POST 'https://api.beta.waroom.com/v1/events' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer <Integration Key>' \
--data-raw '{
    "event": {
        "title": "Test Alert",
        "description": "Test notification triggered",
        "aggregation_key": "test_aggregation_key_1",
        "status": "open"
    }
}'
```

以下の時間JSTあたりに実行しており、uuidについては変化しているので何かしらは作成されているのかなと思います。
- 2022-01-19 13:58:32
- 2022-01-19 14:14:25
- 2022-01-20 09:06:12

画面から手動でincidentを作成した時は、Unresolveに表示されました。

**Comment by sawa-zen** [comment]
ありがとうございます！こちらすぐに確認いたします🙏🏼

**Comment by sawa-zen** [comment]
@redfoxnodareiki
現在原因を調査中ですが、Waroom 全体で既に使用されてしまっている `aggregation_key` の場合にインシデントが作成されていないようです 🙇🏻 
お急ぎで試されたい場合は `aggregation_key` をかぶることが無さそうな値に変更して試していただけますとおそらく作成されるかと思います 🙏🏻

**Comment by redfoxnodareiki** [comment]
@sawa-zen
ありがとうございます！変更したら表示されるようになりました。

**Comment by sawa-zen** [comment]
@redfoxnodareiki 
こちら修正いたしました 🙏🏻 
同じ `aggregation_key` でも 別 `Service` であればそれぞれインシデントが作成されるようになりました :sparkles:
ご報告ありがとうございました 😊
